Configurations/unix-Makefile.tmpl

```diff
@@ -964,12 +964,6 @@ lint:
 generate_apps:
 	( cd $(SRCDIR); $(PERL) VMS/VMSify-conf.pl \
 				< apps/openssl.cnf > apps/openssl-vms.cnf )
-	@ : {- output_off() if $disabled{apps}; "" -}
-	( b=`pwd`; cd $(SRCDIR); \
-	  $(PERL) -I$$b apps/progs.pl -H $(APPS_OPENSSL) > apps/progs.h )
-	( b=`pwd`; cd $(SRCDIR); \
-	  $(PERL) -I$$b apps/progs.pl -C $(APPS_OPENSSL) > apps/progs.c )
```

Comment:
> Aha, apps/progs.c must be made before apps/progs.h... of course. 

Why wasn't this the case before?

Reply:
Because there was always an old progs.c present. It would have failed in a similar way the first time it was run, i.e. this was a bootstrap problem.

Reply:
Good point! Thanks!

Should we ping a Perl-savy reviewer to get this approved?

I can do test batteries but I don't feel comfortable approving as I don't understand all the changes!

Maybe @mspncp can help?

Reply:
Seeing my name mentioned near the word "Perl-savy" is a previously unknown experience... ;-).  I'm not sure whether I'll have the resources  this weekend to look at this pr, too.